Libraries/Image/RCTUIImageViewAnimated.m

```diff
@@ -89,12 +89,12 @@ - (void)setImage:(UIImage *)image
     return;
   }
 
-#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
+#if TARGET_OS_OSX
+  super.image = image;
```

Comment:
May as well call the setter while we're adding new code here? `[super setImage:image];` That way if ever the setter needs to get overridden for additional logic we'll just get it for free.

Reply:
@HeyImChris, I am not sure if I understand. Isn't `[super setImage:image]` the same as `super.image = image`?

Reply:
Yea good question- there are subtle differences between dot notation and bracket notation. A couple of them are
1) dot notation leads to intention ambiguity around how C accesses data from a struct vs. Obj-C dot notation
2) it can be compiler errors in identical lines of code just based off ambiguous object types (e.g. `view.frame.size.width = 30` is a compiler error on Obj-C if `view` is an NSView/UIView but is perfectly valid in C++ or C if view is actually a struct).

[This article](https://www.bignerdranch.com/blog/dot-notation-syntax/) goes into better depth than I'm probably describing. The differences are usually minor fwiw but it's at least a good habit and brings more clarity to the code for when the practical issues do happen to apply :)

Reply:
they are the same. But in ObjC, our team has been using explicit setter calls to make it easier for search code to identify setter vs getter. the dot notation might simplifies things but in practice it hide some ambiguities like struct vs property: ex) view.frame.size.width

Reply:
Yea again it's super minor. Just something that our org considers a slight code improvement, but if there's a good reason not to (like staying consistent with existing calls in this file) then I think it's fine as is too and shouldn't impact merging into master :)

Reply:
no worries. be happy.

Reply:
done